params/index.yaml

```diff
@@ -27,3 +27,16 @@ BlockRoot:
   description: |
     Block root.
     \<hex encoded blockRoot with 0x prefix\>.
+SkipRandaoVerification:
```

Comment:
This is really only relevant for block v3 as other two APIs are deprecated already, and will be removed in next hard fork.

Although we can still update the description for all APIs I'd prefer not to have this separate from the API description, it's less friendly (and visible) for people reading the yaml files directly, and not the rendered explorer.

Reply:
It's feel better to define all params here, there is no rule how much repetition to remove. While reading through the YAML files, if one is using some good IDE, it's usually one click effort to navigate.